VideoLocker/res/values/strings.xml

```diff
@@ -83,14 +83,16 @@
     <string name="dropdown_list_prefix">Dropdown list.</string>
 
     <!-- Navigation Drawer Labels -->
+    <!-- The accessibility description for the navigation header -->
+    <string name="navigation_header">Currently signed in as {username}, {email}</string>
```

Comment:
@catong The string announce when focused on user's name and email in nav menu header.